src/components/Home.js

```diff
@@ -7,14 +7,29 @@ class Home extends Component {
     };
   }
 
+  renderBlocks() {
```

Comment:
Are you planning on populating this function? If not, let's just get rid of that so the page is a bit tidier. Otherwise, looks good!